.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.10", "3.11"]
+        python-version: ["3.8", "3.10", "3.11"]
```

Comment:
Should 3.12 be added to the list?

Reply:
Yes, I will add it to a new PR that I have coming